src/sst/core/Makefile.am

```diff
@@ -298,7 +299,7 @@ sstinfo_x_LDFLAGS = \
 	-export-dynamic \
 	$(SST_LTLIBS_ELEMLIBS)
 
-include tinyxml/Makefile.inc
+include ../../../external/tinyxml/Makefile.inc
```

Comment:
I think you want to use $(top_srcdir) here, not relative in case someone does an out of source build.